### ISSUE TITLE
Links malformed in JSON response

### ISSUE THREAD

**Issue by skiarn** [aws/aws-lambda-go#177]
Hello, I have problems returning json payloads containing urls, the encoding/json Marshal will escape & to unicode format. 

Is there a reason why lambda should escape the characters to unicode? 
If not then its better to use json Encoder and turn on SetEscapeHTML(true) or at least make it accessible to configure.

Or am i missing the correct way to format my payload?

My use-case is returning a pre signed s3 url in a json response payload through the lambda but I cant find a way to do that today.

**Comment by bmoffatt** [comment]
Thanks for submitting #178! 

In the meantime, you could implement custom json encoding/decoding for your function by implementing [lambda.Handler](https://github.com/aws/aws-lambda-go/blob/08d251dc58f204411d79012fd6f1beefb7107fea/lambda/handler.go#L14-L16) and passing to [lambda.StartHandler](https://github.com/aws/aws-lambda-go/blob/08d251dc58f204411d79012fd6f1beefb7107fea/lambda/entry.go#L44-L50), which allows receiving and returning raw bytes

``` go
type Handler struct {
  // ...
}
func (h *Handler) Invoke(ctx context.Context, request []byte) ([]byte, error) {
  // ...
}

func main() {
  lambda.StartHandler(Handler)
}
```

**Comment by bmoffatt** [comment]
implemented an option for this in #444
